pkgs/development/tools/misc/saleae-logic-2/default.nix

```diff
@@ -17,6 +26,9 @@ appimageTools.wrapType2 {
       ''
         mkdir -p $out/etc/udev/rules.d
         cp ${appimageContents}/resources/linux/99-SaleaeLogic.rules $out/etc/udev/rules.d/
+        mkdir -p $out/share/pixmaps
+        ln -s ${desktopItem}/share/applications $out/share/
+        cp ${appimageContents}/usr/share/icons/hicolor/256x256/apps/Logic.png $out/share/pixmaps/Logic.png
```

Comment:
We have a nice setupHook called copyDesktopItems which does exactly this. It should be used instead.

Reply:
please see above conversation